lib/msal-common/src/utils/Constants.ts

```diff
@@ -34,14 +36,26 @@ export const Constants = {
     S256_CODE_CHALLENGE_METHOD: "S256",
     URL_FORM_CONTENT_TYPE: "application/x-www-form-urlencoded;charset=utf-8",
     AUTHORIZATION_PENDING: "authorization_pending",
-    NOT_DEFINED: "not_defined"
+    NOT_DEFINED: "not_defined",
+    // Default time to throttle RequestThumbprint in milliseconds
+    DEFAULT_THROTTLE_TIME_MS: 120000,
+    DEFAULT_MAX_THROTTLE_TIME_MS: 3600000
 };
 
 /**
  * Request header names
  */
 export enum HeaderNames {
-    CONTENT_TYPE = "Content-Type"
+    CONTENT_TYPE = "Content-Type",
+    RETRY_AFTER = "Retry-After",
+    X_MS_LIB_CAPABILITY = "x-ms-lib-capability"
+}
+
+/**
+ * Header values
+ */
+export enum HeaderValues {
```

Comment:
Does this need to be an enum? Are there going to be more items under this?

Reply:
I ended up just putting this with the enum above it, HeaderNames. We can extract separate it for readability if needed